pkg/converter/converter.go

```diff
@@ -493,10 +520,26 @@ func ComposeServiceToK8s(ref string, workload *ir.Service, projectVolumes map[st
 		refSlug = "-" + refSlug
 	}
 
-	composeService := workload.AsCompose()
-
 	objects := Objects{}
 
+	if util.Converter(workload.Labels()) != nil {
+		otherResource, err := CallExternalConverter(workload.Name+refSlug, util.SubConfig(workload.Labels(), "k8ify.converter", "cmd"))
+		if err != nil {
+			log.Fatal(err)
+		}
+		if otherResource.GetLabels() == nil {
+			otherResource.SetLabels(labels)
```

Comment:
Might make sense to _always_ set our custom labels. Otherwise, as soon as the converter adds some other, unrelated labels, users will have to care about figuring out `refSlug` etc. values themselves.

Reply:
The labels are already always set. This is the case where the "otherResource" does not have a labels map. The else case just one line below handles the alternative case where "otherResource" already has a labels map, in which case we have to merge the two.